src/component/feature/FeatureView/FeatureView.tsx

```diff
@@ -133,6 +133,9 @@ export const FeatureView = () => {
         );
     };
 
+    if (error?.status === 404) {
+        return renderFeatureNotExist();
```

Comment:
We also need to check if the feature exists in archive here. Because the feature might exist and be archived.

Reply:
@FredrikOseberg Done ✅